packages/react/spec/auto/inputs/PolarisDateTimePicker.spec.tsx

```diff
@@ -16,17 +16,49 @@ describe("PolarisDateTimePicker", () => {
       ...originalDateResolvedOptions,
       timeZone: "America/New_York",
     });
+
     const onChangeSpy = jest.fn();
-    const user = userEvent.setup();
-    const baseDate = new Date("2021-03-05T11:23:10.000Z");
-    render(<PolarisDateTimePicker id="test" includeTime value={baseDate} onChange={onChangeSpy} />, { wrapper: PolarisWrapper });
+    render(
+      <PolarisAutoForm action={api.widget.create} findBy={"42"}>
+        <PolarisDateTimePicker id="test" includeTime hideTimePopover onChange={onChangeSpy} field="startsAt" />
+        <PolarisAutoSubmit />
+      </PolarisAutoForm>,
+      { wrapper: PolarisMockedProviders }
+    );
 
+    mockUpdateWidgetFindBy();
+    const user = userEvent.setup();
     await act(async () => {
       await user.click(screen.getByLabelText("Time"));
-      await user.clear(screen.getByLabelText("Time"));
-      await user.type(screen.getByLabelText("Time"), "11:00 AM");
+      await user.type(screen.getByLabelText("Time"), "11");
+      await user.type(screen.getByLabelText("Time"), ":");
+      await user.type(screen.getByLabelText("Time"), "00 ");
+      await user.type(screen.getByLabelText("Time"), "AM");
+      await user.click(screen.getByRole("button"));
     });
 
-    expect(onChangeSpy).toHaveBeenCalledWith(new Date("2021-03-05T16:00:00.000Z"));
+    const formSubmitRequest = mockUrqlClient.executeMutation.mock.calls[0][0];
+    const startsAt = formSubmitRequest.variables.widget.startsAt;
+    expect(startsAt).toEqual(today);
+    expect(onChangeSpy).toHaveBeenCalledWith(today);
   });
 });
+
+const today = new Date();
+today.setHours(15);
+today.setMinutes(0);
+today.setSeconds(0);
+today.setMilliseconds(0);
```

Comment:
This area ends up getting called before the call to 
```
jest.spyOn(Intl.DateTimeFormat.prototype, "resolvedOptions").mockReturnValue({
```
I don't that makes a difference now considering CI is green, but could you move this area below that statement so that we don't need to think about this in the event we need to revisit this area?